packages/extension-template-ts/src/index.ts

```diff
@@ -0,0 +1,55 @@
+import { JsPsych, JsPsychExtension, JsPsychExtensionInfo } from "jspsych";
+
+interface InitializeParameters {
+
+}
+
+interface OnStartParameters {
+
+}
+
+interface OnLoadParameters {
+
+}
+
+interface OnFinishParameters {
+
+}
+
+/**
+ * **EXTENSION-NAME**
+ *
+ * SHORT PLUGIN DESCRIPTION
+ *
+ * @author YOUR NAME
+ * @see {@link https://DOCUMENTATION_URL DOCUMENTATION LINK TEXT}
+ */
+class ExtensionNameExtension implements JsPsychExtension {
+  static info: JsPsychExtensionInfo = {
+    name: "extension-name"
+  }
+
+  constructor(private jsPsych: JsPsych) {}
+
+  initialize = ({}:InitializeParameters): Promise<void> => {
+    return new Promise((resolve, reject)=>{
```

Comment:
This is showing an error in VS Code because it says Promise is only a type. Not sure what the root cause of that is.

Reply:
Not happening for me. No idea either. Does it work when you run `npm install` in the repo root?

Reply:
That fixed it. I discovered that the other day when you mentioned I hadn't run it in root. I forgot to update this